source/fundamentals/connection/connect.txt

```diff
@@ -39,16 +39,16 @@ reconfiguring clients.
    To learn how to retrieve your connection string in Atlas, see the
    :atlas:`Atlas driver connection guide </driver-connection>`.
 
-The next part of the connection string contains your username and password
+The next part of the connection string contains your credentials
 if you are using password-based authentication. Replace the value of ``user``
 with your username and ``pass`` with your password. If you are using an
 authentication mechanism that does not require a username and password, omit
 this part of the connection URI.
 
-The next part of the connection string specifies the hostname or IP address and
-port of your MongoDB instance. In the example above, we use ``sample.host``
-as the hostname and ``27017`` as the port. Replace these values to point to
-your MongoDB instance.
+The next part of the connection string specifies the hostname or IP address,
+followed by the port of your MongoDB instance. In the example above, we use
```

Comment:
I: This is a good change, but it introduces a new (possible) ambiguity: Does my MongoDB instance have a hostname/IP address as well as a port? Or does it just have a port?

S: 
```suggestion
The next part of the connection string specifies the hostname or IP address
of your MongoDB instance, followed by the port number. In the example above, we use
```

Happy to workshop this